### PR TITLE
Manifest zeus: Update  and add connagtive-qt5demo-image for sdk build

### PR DESCRIPTION
**Pull request by motto-phytec**

### CODE REVIEW

File: zeus.xml [hunk 1]
```diff
@@ -43,7 +49,7 @@
     <sublayer path="meta-gnome"/>
   </project>
 
-  <project name="meta-imx" path="sources/meta-imx" remote="codeaurora" revision="refs/tags/rel_imx_5.4.70_2.3.2" upstream="zeus-5.4.70-2.3.2">
+  <project name="meta-imx" path="sources/meta-imx" remote="codeaurora" revision="refs/tags/rel_imx_5.4.70_2.3.0" upstream="zeus-5.4.70-2.3.0">
```
Comment (mschwan-phytec) [inline zeus.xml]: Please undo this change. We are going to use version 2.3.2 for the next release.

File: recipes-images/images/phytec-connagtive-qt5demo-image.bb [hunk 2]
```diff
@@ -0,0 +1,28 @@
+require recipes-images/images/phytec-connactive-start-image.bb
```
Comment (mschwan-phytec) [inline recipes-images/images/phytec-connagtive-qt5demo-image.bb]: There is a typo here: "connagtive" with "g" not "c".

**Review by mschwan-phytec — CHANGES_REQUESTED** [review]

File: zeus.xml (hunk 1 above)
Comment (motto-phytec) [reply zeus.xml]: i know. this commit change the version from 2.3.2 to 2.3.0. it was wrong before.

File: recipes-images/images/phytec-connagtive-qt5demo-image.bb (hunk 2 above)
Comment (motto-phytec) [reply recipes-images/images/phytec-connagtive-qt5demo-image.bb]: is changed

**Pull request closed without merging**